flow-decomposition/src/test/java/com/powsybl/flow_decomposition/CgmesIntegrationTests.java

```diff
@@ -63,10 +65,14 @@ void checkFlowDecompositionWorksOnCgmesFile() {
             .setEnableLossesCompensation(FlowDecompositionParameters.ENABLE_LOSSES_COMPENSATION)
             .setLossesCompensationEpsilon(FlowDecompositionParameters.DISABLE_LOSSES_COMPENSATION_EPSILON)
             .setSensitivityEpsilon(FlowDecompositionParameters.DISABLE_SENSITIVITY_EPSILON)
-            .setRescaleEnabled(FlowDecompositionParameters.ENABLE_RESCALED_RESULTS)
-            .setXnecSelectionStrategy(FlowDecompositionParameters.XnecSelectionStrategy.INTERCONNECTION_OR_ZONE_TO_ZONE_PTDF_GT_5PC);
-        FlowDecompositionComputer flowDecompositionComputer = new FlowDecompositionComputer(flowDecompositionParameters);
-        FlowDecompositionResults flowDecompositionResults = flowDecompositionComputer.run(network);
-        assertNotNull(flowDecompositionResults.getDecomposedFlowMap());
+            .setRescaleEnabled(FlowDecompositionParameters.ENABLE_RESCALED_RESULTS);
+        Optional<Branch> optionalBranch = network.getBranchStream().findAny();
```

Comment:
Prefer usage of a fixed ID from CGMES file, this method to get a branch may have unforeseen impact, as choosing a disconnected branch